controllers/argocd/rolebinding_test.go

```diff
@@ -184,6 +184,32 @@ func TestReconcileArgoCD_reconcileClusterRoleBinding(t *testing.T) {
 	assert.NoError(t, r.Client.Get(context.TODO(), types.NamespacedName{Name: expectedName}, clusterRoleBinding))
 }
 
+func TestReconcileArgoCD_reconcileClusterRoleBinding_disabled(t *testing.T) {
+	logf.SetLogger(ZapLogger(true))
```

Comment:
Same as my above in `role_test.go`, good to check the case where:
- `DefaultClusterScopedRoleCreationDisabled = true` -> `DefaultClusterScopedRoleCreationDisabled = false: ensure ClusterRoleBinding is created`
- `DefaultClusterScopedRoleCreationDisabled = false` -> `DefaultClusterScopedRoleCreationDisabled = true: ensure ClusterRoleBinding is deleted`